x-pack/plugins/observability_solution/infra/public/alerting/inventory/index.ts

```diff
@@ -23,29 +23,29 @@ interface InventoryMetricRuleTypeParams extends RuleTypeParams {
 const inventoryDefaultActionMessage = i18n.translate(
   'xpack.infra.metrics.alerting.inventory.threshold.defaultActionMessage',
   {
-    defaultMessage: `\\{\\{context.reason\\}\\}
+    defaultMessage: `'{{context.reason}}'
```

Comment:
Is additional `'` needed?

Reply:
not really 1 should be enough to escape the whole message. so for a message like this with only a single element it should be fine. it is escaping the opening paranthesis so escaping the closing one are not needed. i will add both just for it to look more readable and for consistency.